app/public/stylesheets/sass/_components/_view-container.scss

```diff
@@ -12,11 +12,53 @@
     margin: 0 auto;
   }
 
-  & > div > h1 {
+  & > div > h1, & > div > header > h1 {
     margin: 10px 0 30px;
     font-weight: 300;
     font-size: 3em;
   }
+
+    & > div > header {
```

Comment:
selectors not following name convention e.g `favoritesView_componentName` https://github.com/Soundnode/soundnode-app/wiki/HTML-CSS-convention